ocf/ocf_test.go

```diff
@@ -450,7 +493,6 @@ func TestEncoder_EncodeCompressesDeflate(t *testing.T) {
 
 	buf := &bytes.Buffer{}
 	enc, _ := ocf.NewEncoder(schema, buf, ocf.WithCodec(ocf.Deflate))
-	defer enc.Close()
```

Comment:
The encoder is closed 5 lines down :)

Reply:
No, it is defered closed, so would close at the end of the test. In modern go should now be:

```
t.Cleanup(func() { _ = enc.Close() }
```

Reply:
Nvm, just understood what you meant.

Reply:
And In regard to the cleanup, I did not know that, but I guess you need to Close to trigger the flush. With deferred Close and no flush I guess you will end up with 0 bytes in your buffer. nvm I will mark as resolved, unless you have further comments.